netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -247,6 +249,18 @@ protected NettyServer buildTransportServer() {
   }
 
   @Override
+  public NettyServerBuilder useTransportSecurity(TrustManagerFactory trustManagerFactory,
+                                                 KeyManagerFactory keyManagerFactory) {
+    try {
+      sslContext = GrpcSslContexts.forServer(trustManagerFactory, keyManagerFactory).build();
+    } catch (SSLException e) {
+      // This should likely be some other, easier to catch exception.
```

Comment:
We can shave this yak (copied from original method) while we're here. Any suggestions what should be thrown here?